SSLP_analyzer/SSLP_analyzer/settings.py

```diff
@@ -0,0 +1,130 @@
+"""
+Django settings for SSLP_analyzer project.
+
+Generated by 'django-admin startproject' using Django 4.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+from pathlib import Path
+import os
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.environ.get('DJANGO_SECRET_KEY')
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
For production this is true. Although then you have to start up your own static file server to host the js and css files. So the user can access those.

Reply:
Some possible solutions are given on [stackoverflow](https://stackoverflow.com/questions/5836674/why-does-debug-false-setting-make-my-django-static-files-access-fail). Although likely, Django will only run on local computers, this is open source software so if anybody wants to run it on a web server, they can. It would be good not to require `DEBUG = True` as this is considered very insecure.